cni/installer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang as build
```

Comment:
This is the closest I could comment on this :). There's a busybox blob being checked in which should be removed.